extensions/sample-greeting/deployment/pom.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-sample-greeting-parent</artifactId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>quarkus-sample-greeting-deployment</artifactId>
+    <name>Quarkus - Sample Greeting Extension - Deployment</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-core-deployment</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-sample-greeting</artifactId>
+        </dependency>
+        <dependency>
+          <groupId>io.quarkus</groupId>
+          <artifactId>quarkus-undertow-deployment</artifactId>
```

Comment:
do you need undertow specifically? We try and use it less and less unless you must have servlet API support

Reply:
OK I see you use it in the doc. I'd argue it would be better to have a different enxample than using the legacy servlets but we can go with that initially.

Reply:
@cescoffier got an alternative routing proposal?

Reply:
I started to migrate the sample extension to the quickstart but still interested by the answer 😄